bcipy/parameters/parameters.json

```diff
@@ -744,16 +744,25 @@
     "type": "bool"
   },
   "lang_model_type": {
-    "value": "PRELM",
+    "value": "GPT2",
     "section": "lang_model_config",
     "readableName": "Language Model Type",
-    "helpTip": "Specifies which language model to use. Default: PRELM",
+    "helpTip": "Specifies which language model to use. Default: GPT2",
     "recommended_values": [
+      "GPT2",
       "PRELM",
       "UNIFORM"
     ],
     "type": "str"
   },
+  "lm_path": {
```

Comment:
The path only includes the directory.

Reply:
This parameter has type "str" because the model allows 'gpt2' as the lm_path, otherwise it would be type of "directorypath". Maybe we should change the model so the default value is `None`, and sets it to "gpt2" if missing.